src/components/map-page-components/map-page/MapPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import MapSidebar from '../map-sidebar/MapSidebar';
+import MapView from '../map-view/MapView';
+import './map-page.less';
+import { Layout } from 'antd';
+
+const { Content, Sider } = Layout;
+
+type MapPageProps = {
+  sidebarTitle: string;
+  sidebarDescription: string;
+};
+
+const MapPage: React.FC<MapPageProps> = (props) => {
+  const sideBarWidth = 471;
```

Comment:
Hmm, how are we going to do this long-term? How will this interact with mobile?

Reply:
I suppose that with media queries the sidebar would just not exist on the mapPage on mobile, the bar is collapsible, has different margins, no description, etc. so it might make more sense to just have a different component on mobile instead of trying to make a hamfisted sidebar and footer with a ton of variables.

Reply:
That also might just mean we want the sidebar itself within the sidebar component instead of on the MapPage level